install.sh

```diff
@@ -238,6 +239,13 @@ __install_zeek() {
                 "security:zeek" \
                 "https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
                 ;;
+            Debian)
+                __install_packages cmake make gcc g++ flex bison libpcap-dev libssl-dev python3 python3-dev python3-git python3-semantic-version swig zlib1g-dev gpg
```

Comment:
I know these dependencies are required to build Zeek. However, I don't believe these are required to install Zeek from a deb package.

Reply:
I do not mind at all! After testing with the line removed, I needed to add gpg back as a required dependency.